v0/destinations/splitio/config.js

```diff
@@ -0,0 +1,32 @@
+const { getMappingConfig } = require("../../util");
+
+const BASE_URL = "https://events.split.io";
+
+const endpoints = {
+  eventUrl: `${BASE_URL}/api/events`,
+  identifyUrl: `${BASE_URL}/internal/api/v2/users `,
+  GroupUrl: `${BASE_URL}/internal/api/v2/groups`
+};
+
+const CONFIG_CATEGORIES = {
+  EVENT: { endPoint: endpoints.eventUrl, name: "EventConfig" },
+  IDENTIFY: { endPoint: endpoints.identifyUrl, name: "IdentifyConfig" },
```

Comment:
no need for this and group as we are only using /events. above also remove. 
Not using the other api as we do not get the user id to check if user exists or not thus we cannot add every time as error will arise.

Reply:
removed identify and group as endpoints and also from config categories.